storybook/src/components/IconButton/IconButton.docs.mdx

```diff
@@ -34,16 +34,11 @@ import README from "../../../../packages/css/src/components/icon-button/README.m
 
 An Icon Button on a coloured background must adjust its foreground colour to provide enough contrast.
 We have lighter and darker background colours, and Icon Buttons behave differently on each.
-Stijlweb shows [which colours suit which backgrounds](https://amsterdam.nl/stijlweb/basiselementen/kleuren/).
-Although they only address headings and paragraphs, the same principle applies to Icon Buttons.
 
-- Set the `color` prop to `contrast` on a yellow, orange or green background.
+- Set the `color` prop to `contrast` on a azure, lime, orange or yellow background.
   It will make the Icon Button black.
-- Set the `color` prop to `inverse` on a purple or dark blue background.
+- Set the `color` prop to `inverse` on a blue, green, magenta, purple or red background.
```

Comment:
It's currently not possible to have an IconButton on a red background, no? Same for Link and LinkList.